### ISSUE TITLE
Connection timeout to webui of deluge

### ISSUE THREAD

**Issue by skay973** [sebgl/htpc-download-box#4]
Hi

First of all, very good job for this tutorial ! very complete and detailed, awesome !

I'm having an issue after the setup of the vpn. I use a PIA account like in your example but when trying to access to the web ui from my personal computer, I get a connection time out. Looks like its trying to reach during 5secs and then response with a timeout message. Im note sure where to investigate, is it a server issue, deluge web ui issue or vpn issue ? 

Cheers

**Comment by sebgl** [comment]
Can you give me a bit more details:
- what does your setup look like? Is everything running on your personal computer? Or do you run these on a distant server?
- which "web ui" are you talking about? Deluge?
- one thing to check is whether the VPN connection succeeded or not. You can have a look at the VPN container logs for that (`docker logs vpn`)

**Comment by skay973** [comment]
Hi !

Thanks for your response,

1) Regarding the setup I use a kimsufi (powered by ovh) server, so everything is managed remotely through ssh

2) yes I'm talking about deluge web ui 

3) the vpn is up and running

Quick update now, I think I understood part of the problem, I was trying to reach deluge web ui from my personal computer, so not on the same local network as the server where docker is running. So I connected through ssh tunnel and portforwarding 8112, and I am able to managed deluge from my personal computer now, I did the torguard test check my torrent ip and I can see an IP that's not the one of my kimsufi server. I guess it's working. However, I can see that my deluge is not downloading... I added manually magnet link to test the configuration, web ui says "dowloading", tracker status says OK but nothing is downloading :'(...

Any idea ?

**Comment by sebgl** [comment]
Could be... but I'd say that's unlikely. AFAIK both OVH and PIA do not restrict bittorent usage.
Have you tried a very popular torrent with a lot of sources, to eliminate the fact that maybe your torrent has poor seeds/peers?

Also you can check if the deluge container still has access to the internet when it's not downloading:
```
docker exec -ti deluge bash
curl google.com
```

**Comment by csandman** [comment]
@skay973 If you remember, can you give some more details on what you did to fix the web ui? I'm running into the same issue but haven't had any luck with it.  When you say port forwarded it, do you mean your router, or in the docker config?

**Comment by csandman** [comment]
I finally figured out how to prevent this timeout.  Its been bugging me for weeks.

This whole time, I've been trying to connect to it using 10.0.0.2:8112 (my server's static local ip) from my laptop.  Instead I realized I need to port forward that port through ssl ON MY LAPTOP.

`ssh -NL 8112:localhost:8112 <server ip>`

which for me is:

`ssh -NL 8112:localhost:8112 chris@10.0.0.2`

and THAT allows me to connect at `localhost:8112` from my laptop.  This may be obvious to many of you but I built my first ubuntu server about 3 weeks ago so this is my first time really having to deal with any sort of networking.

I hope this helps if someone has the same problem!
